src/ballet/txn/fd_txn.h

```diff
@@ -386,8 +386,12 @@ fd_txn_footprint( ulong instr_cnt,
    accumulated into the struct pointed to by counters_opt. Note: The returned
    txn object is not self-contained since it refers to byte ranges inside the
    payload. */
-ulong fd_txn_parse( uchar const * payload, ulong payload_sz, void * out_buf, fd_txn_parse_counters_t * counters_opt );
-
+ulong fd_txn_parse( uchar const             * payload, 
+                    ulong                     payload_sz, 
+                    void *                    out_buf, 
+                    fd_txn_parse_counters_t * counters_opt,
+                    ulong                   * bytes_parsed_sz_opt );
+              
```

Comment:
Could we break the txn changes, the bc_types, and the binparse into its own PR so that we can get it merged into main ASAP?